easybuild/easyblocks/b/bamtools.py

```diff
@@ -58,10 +58,22 @@ def sanity_check_step(self):
         sharedlib_ext = get_shared_lib_ext()
 
         custom_paths = {
-            'files': ["bin/bamtools", "include/shared/bamtools_global.h", "lib/libbamtools.a",
-                      "lib/libbamtools.%s" % sharedlib_ext, "lib/libbamtools-utils.%s" % sharedlib_ext,
-                      "lib/libjsoncpp.%s" % sharedlib_ext],
-            'dirs': ["include/api", "docs"]
+            'files': [
+                "bin/bamtools",
+                "include/shared/bamtools_global.h",
+                "lib/libbamtools.a",
+                "lib/libbamtools.%s" % sharedlib_ext],
```

Comment:
I've started using pylint and it didn't warn on this.

I'm not sure whether it is better or not really. I guess it is a question of whether you prefer array items to be aligned as close to the left as possible.

I can fix it, but I'd need to fix more than this one in the file to be consistent.